src/Config/ObjectModel/EntityRestOptions.cs

```diff
@@ -18,5 +18,8 @@ namespace Azure.DataApiBuilder.Config.ObjectModel;
 [JsonConverter(typeof(EntityRestOptionsConverter))]
 public record EntityRestOptions(SupportedHttpVerb[] Methods, string? Path = null, bool Enabled = true)
 {
+    public const string PROPERTY_PATH = "path";
+    public const string PROPERTY_METHODS = "methods";
+    public const string PROPERTY_ENABLED = "enabled";
```

Comment:
Is there an alternative approach to manually creating properties for each record property? i.e. similar to what we had in the past, [using jsonpropertyname attribute](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record#:~:text=You%20may%20want,to%20the%20parameter.):
```csharp
/// <summary>
/// Person record type
/// </summary>
/// <param name="FirstName">First Name</param>
/// <param name="LastName">Last Name</param>
/// <remarks>
/// The person type is a positional record containing the
/// properties for the first and last name. Those properties
/// map to the JSON elements "firstName" and "lastName" when
/// serialized or deserialized.
/// </remarks>
public record Person([property: JsonPropertyName("firstName")] string FirstName, 
    [property: JsonPropertyName("lastName")] string LastName);
```

Reply:
This approach helped in deserializing and to help know the mapping from `firstName` in json to `FirstName` in object. Essentially we cannot refer to the jsonpropertyname anywhere. Like in EntityRestOptionsConverter.cs class since it is an attribute.